src/components/organisms/TimingEventModal/TimingEventModal.scss

```diff
@@ -40,4 +40,25 @@
   .ButtonNG {
     max-width: none;
   }
+
+  .AdminDateTime {
+    margin-top: 0;
+    margin-bottom: 0.8rem;
+  }
+
+  .AdminDateTime__input-wrapper {
+    padding: 0;
+    input {
+      border: 0.1px solid var(--accent--over-20a);
+    }
+  }
+
+  // This is to combat the use of a h2 tag in the wrong place. Longer term
```

Comment:
Why is `h2` in the wrong place?
Does the page have no `h1`?
Does the `h2` come before or after some other `h` tags that make it inappropriate?

Reply:
It's basically a label in a form. Visually it looks wrong and not like the rest of the form.